build/postgres/Dockerfile

```diff
@@ -82,7 +81,6 @@ else \
 		python3-psutil \
 		python3-psycopg2 \
 		rsync \
-		$( printf '11\n'${PG_MAJOR} | sort -VC && echo timescaledb_${PG_MAJOR} ) \
```

Comment:
🤔 I know these aren't easy to comprehend. It's unfortunate we can't comment around here, either. This was doing "11 <= PG_MAJOR."

📝 We could update it to "11 <= PG_MAJOR <= 14" with 
```sh
printf '11\n'${PG_MAJOR}'\n14' | sort -VC
```

or to "10 < PG_MAJOR < 15" with
```sh
printf '10\n'${PG_MAJOR}'\n14' | sort -uVC
```

I prefer the former.

Reply:
Oh wow, I didn't realize this is what we were doing. A comment here would have been helpful. I can update to use this logic instead of a new IF statement

Reply:
🙇🏻 ⛏️ I think I'm finding I more than "prefer" using `11` and `14`. I read it as "for 11 through 14, timescale." I'm not sure how to read `10` and `15`... "for everything but 10 ... or 15 ..."

It's a nitpick at this point. The logic is sound.